2022-12-02 14:46:42.5090 ERROR Something went wrong: ProjectManagementTrackerAPI.BL.InvalidProjectAndTaskEndDate: Invalid Project nad task end date
   at ProjectManagementTrackerAPI.BL.Validation.ValidateProjectAndTaskEnddate(DateTime projectEndDate, DateTime taskEndDate) in D:\Local_Project\ProjectManagementTracker\ProjectManagementTracketAPI\BL\Validation.cs:line 38
   at ProjectManagementTracketAPI.Repository.MemberRepository.AssigningTask(AssigningTaskDTO assigningTaskDTO) in D:\Local_Project\ProjectManagementTracker\ProjectManagementTracketAPI\Repository\MemberRepository.cs:line 85
   at ProjectManagementTracketAPI.Controllers.V1.ManagerController.AssignTask(AssigningTaskDTO assigningTaskDTO) in D:\Local_Project\ProjectManagementTracker\ProjectManagementTracketAPI\Controllers\V1\ManagerController.cs:line 68
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2022-12-02 14:48:32.5639 ERROR Something went wrong: ProjectManagementTrackerAPI.BL.InvalidProjectAndTaskEndDate: Invalid Project nad task end date
   at ProjectManagementTrackerAPI.BL.Validation.ValidateProjectAndTaskEnddate(DateTime projectEndDate, DateTime taskEndDate) in D:\Local_Project\ProjectManagementTracker\ProjectManagementTracketAPI\BL\Validation.cs:line 38
   at ProjectManagementTracketAPI.Repository.MemberRepository.AssigningTask(AssigningTaskDTO assigningTaskDTO) in D:\Local_Project\ProjectManagementTracker\ProjectManagementTracketAPI\Repository\MemberRepository.cs:line 85
   at ProjectManagementTracketAPI.Controllers.V1.ManagerController.AssignTask(AssigningTaskDTO assigningTaskDTO) in D:\Local_Project\ProjectManagementTracker\ProjectManagementTracketAPI\Controllers\V1\ManagerController.cs:line 68
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
